Avoid building the parts we don't use that drag in extra dependencies.

Index: cabal.project
--- cabal.project.orig
+++ cabal.project
@@ -1,21 +1,5 @@
-with-compiler: ghc-9.8.2
-index-state: 2024-07-03T13:49:27Z
-tests: True
-
 packages: cabal-bundler
-packages: cabal-core-inspection
-packages: cabal-deps
-packages: cabal-diff
-packages: cabal-docspec
-packages: cabal-env
-packages: cabal-haddock-server
-packages: cabal-hasklint
-packages: cabal-hie
-packages: cabal-iface-query
 
-packages: cabal-store-check
-packages: cabal-store-gc
-
 packages: peura
 
 -- compatibility with new directory
@@ -48,7 +32,6 @@ constraints: transformers  installed
 
 -- For local dev these can be used:
 packages: extras/gentle-introduction-*.tar.gz
-packages: extras/hooglite-*.tar.gz
 
 -- Always use ghc-lib-parser.
 constraints: ghc-lib-parser-ex -no-ghc-lib -auto
@@ -56,8 +39,6 @@ constraints: ghc-lib-parser-ex -no-ghc-lib -auto
 -- Never depend on cryptonite. Aptly named package.
 constraints: cryptonite <0
 
--- Warp without x509
-constraints: warp -x509
 
 allow-newer: hooglite:Cabal
 
